README.md

```diff
@@ -64,185 +64,266 @@ for an example.
+            
+            For a struct in Uniform, `StorageClass`, or `PushConstant`: - a member Offset must be a multiple of the member's scalar alignment - `ArrayStride` or `MatrixStride` must be a multiple of the array or matrix scalar alignment
+  
         --skip-block-layout
-            Skip checking standard uniform / storage buffer layout. Overrides any --relax-block-layout or --scalar-block-layout option
-
+            Records whether or not the validator should skip validating standard uniform/storage block layout
+  
         --preserve-bindings
-            Preserve unused descriptor bindings. Useful for reflection
-
+            Records whether all bindings within the module should be preserved
+  
+    -m, --manifest-file <MANIFEST_FILE>
+            Renames the manifest.json file to the given name
+            
+            [default: manifest.json]
+  
     -h, --help
             Print help (see a summary with '-h')
 
 
 * Show
   Show some useful values
-
+  
   Usage: cargo-gpu show <COMMAND>
-
+  
   Commands:
     cache-directory  Displays the location of the cache directory
     spirv-source     The source location of spirv-std
+    commitsh         The git commitsh of this cli tool
+    capabilities     All the available SPIR-V capabilities that can be set with `--capabilities`
     help             Print this message or the help of the given subcommand(s)
-
+  
   Options:
     -h, --help
             Print help
 
 
         * Cache-directory
           Displays the location of the cache directory
-
+          
           Usage: cargo-gpu show cache-directory
-
+          
           Options:
             -h, --help
                     Print help
 
 
         * Spirv-source
           The source location of spirv-std
-
+          
           Usage: cargo-gpu show spirv-source [OPTIONS]
-
+          
           Options:
                 --shader-crate <SHADER_CRATE>
                     The location of the shader-crate to inspect to determine its spirv-std dependency
-
+                    
                     [default: ./]
+          
+            -h, --help
+                    Print help
+
+
+        * Commitsh
+          The git commitsh of this cli tool
+          
+          Usage: cargo-gpu show commitsh
+          
+          Options:
+            -h, --help
+                    Print help
 
+
+        * Capabilities
+          All the available SPIR-V capabilities that can be set with `--capabilities`
+          
+          Usage: cargo-gpu show capabilities
+          
+          Options:
```

Comment:
I wish there were a way we could keep the usage in the README in sync with the tool without having to manually insert it here. 🤔

Reply:
Thank you for doing it, though, obviously.